usaspending_api/database_scripts/cleaners/purge_legal_entities.sql

```diff
@@ -10,17 +10,15 @@ WHERE legal_entity_id IN (
     FROM legal_entity le
     WHERE (
         NOT EXISTS (SELECT 1 FROM transaction_normalized WHERE recipient_id = le.legal_entity_id) AND
-        NOT EXISTS (SELECT 1 FROM awards WHERE recipient_id = le.legal_entity_id) AND
-        NOT EXISTS (SELECT 1 FROM subaward WHERE prime_recipient_id = le.legal_entity_id)
+        NOT EXISTS (SELECT 1 FROM awards WHERE recipient_id = le.legal_entity_id)
     )
 );
 
 DELETE
 FROM legal_entity le
 WHERE (
     NOT EXISTS (SELECT 1 FROM transaction_normalized WHERE recipient_id = le.legal_entity_id) AND
-    NOT EXISTS (SELECT 1 FROM awards WHERE recipient_id = le.legal_entity_id) AND
-    NOT EXISTS (SELECT 1 FROM subaward WHERE prime_recipient_id = le.legal_entity_id)
```

Comment:
What are these lines doing?

Reply:
I believe they're removing obsolete legal entities.

Reply:
Does this PR do anything with the legal_entities table?

Reply:
No.

Reply:
Then should this be here?
What I was originally trying to ask was "why did we make these changes; they don't seem related"

Reply:
Because `prime_recipient_id` was removed from the `subaward` table.